examples/8.0/README.md

```diff
@@ -22,8 +22,8 @@ Verification commands
 Run the following commands to validate things are rolling as they should.
 
 ```bash
-# Should use 8.0.39 as the default version
-lando exec defaults -- mysql --version | grep 8.0.39
+# Should use 8.4 as the default version
```

Comment:
Let's make a distinct 8.4 example; not much more work and maintains coverage of 8.0.